jenkinsapi/utils/crumb_requester.py

```diff
@@ -16,6 +16,26 @@ def __init__(self, *args, **kwargs):
         self._baseurl = kwargs['baseurl']
         self._last_crumb_data = None
 
+    def get_url(self, url, params=None, headers=None, allow_redirects=True,
+                stream=False):
+        if self._last_crumb_data:
+            # first try request with previous crumb if available
+            response = self._get_url_with_crumb(
+                self._last_crumb_data, url, params,
+                headers, allow_redirects, stream
+            )
+            # code 403 might indicate that the crumb is not valid anymore
+            if response.status_code != 403:
+                return response
+
+        # fetch new crumb (if server has crumbs enabled)
+        if self._last_crumb_data is not False:
```

Comment:
This logic is copied from the existing post_url implementation for consistency. This check is here in case someone mistakenly uses the CrumbRequester for a Jenkins server which doesn't have CSRF enabled. If it is the case, no crumb data is retrieved and the `_get_crumb_data()` method [returns False](https://github.com/MichaelPereira/jenkinsapi/blob/f7a7c8f12ea200ac231dfe7135a9ef38e6587cea/jenkinsapi/utils/crumb_requester.py#L63).